### ISSUE TITLE
Add github issue template to encourage information gathering.

### ISSUE THREAD

**Issue by serundeputy** [backdrop/backdrop-issues#2676]
Issue templates can help ensure that each issue contains enough information to investigate, solve and work on the requested issue.

-----
PR: https://github.com/backdrop/backdrop/pull/1879

**Comment by klonos** [comment]
Usually when starting to describe and issue, I first start saying things like "When you navigate to ... and click on...". These are STR.

I then either explain verbally or usually attach a screenshot of the outcome. Then I explain what's wrong with the screenshot if it is not already obvious (I use a lot of arrows, shapes and inline text in my screenshots).

I then move on to either verbally explain expected behavior or provide mockup screenshots.

In some cases (usually after a request from anther person), I return to add extra info such as Backdrop version or operating system or browser etc.

So, I think that the natural order would be more like:

### Steps to reproduce

### Actual behavior

### Expected behavior

### Any additional version/system information

**Comment by serundeputy** [comment]
updated PR to @klonos template:

### Steps to reproduce

### Actual behavior

### Expected behavior

### Any additional version/system information

**Comment by Dinornis** [comment]
Sorry, I am old and confused 😁 

I understand a need for a template for Backdrop users to guide them when filing a new issue here at GitHub, this will be very helpful.  Where will this template be posted and what is the purpose of the PR linked to this issue?

**Comment by serundeputy** [comment]
@Dinornis once the template is merged each issue opened will be pre populated with these prompts.

**Comment by serundeputy** [comment]
I realized we don't need this PR against core; I've merged it to backdrop/backdrop-issues 9aa375b0a3ca7809c4dbeb670186689cde26ceb5 so the template is in place and easy to change should we need/want to. :)
